lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -78,11 +79,18 @@ if($tags.count){
 }
 # Run Pester tests
 If (Test-Path -LiteralPath $path -PathType Leaf) {
+    if ($testParameters.keys.count) {
+        $Parameters.Script = @{Path = $Path ; Parameters = $testParameters }
+        $testParametersCheckModeMsg = "with $($testParameters.keys -join ',') parameters"
+    }
+    else {
+        $Parameters.Script = $Path
+    }
     if ($check_mode) {
-        $result.output = "Run pester test in the file: $path"
+        $result.output = "Run pester test in the file: $path $testParametersCheckModeMsg"
```

Comment:
The `$testParametersCheckModeMsg` variable won't be defined if there are not entries in `test_parameters`. This should be set to an empty string before your `if ($testParameters.keys.count)` check and the space between `$path` and this var removed.

Reply:
> The `$testParametersCheckModeMsg` variable won't be defined if there are not entries in `test_parameters`

In that case its an undefined variable and defaults to nothing, but of course I'll change that.